agent/api/ecsclient/client.go

```diff
@@ -305,25 +331,12 @@ func (client *APIECSClient) SubmitTaskStateChange(change api.TaskStateChange) er
 		return nil
 	}
 
-	var attachments []*ecs.AttachmentStateChange
-
-	if change.Attachments != nil {
-		eniStatus := change.Attachments.Status.String()
-		attachments = []*ecs.AttachmentStateChange{
-			{
-				AttachmentArn: &change.Attachments.AttachmentArn,
-				Status:        &eniStatus,
-			},
-		}
-	}
-
 	taskStatus := change.Status.String()
 	_, err := client.submitStateChangeClient.SubmitTaskStateChange(&ecs.SubmitTaskStateChangeInput{
-		Cluster:     &client.config.Cluster,
-		Task:        &change.TaskArn,
-		Status:      &taskStatus,
-		Reason:      &change.Reason,
-		Attachments: attachments,
+		Cluster: &client.config.Cluster,
```

Comment:
Can you use `aws.String` instead of pointers here (and for other parameters as well)?